### PR TITLE
feat: add data feeds for Gearbox

### PR DESCRIPTION
**Pull request by cehali**

### CODE REVIEW

File: packages/oracle-node/src/fetchers/ccxt/CcxtFetcher.ts [hunk 1]
```diff
@@ -66,17 +66,16 @@ export class CcxtFetcher extends BaseFetcher {
       throw new Error(`Price not returned for: ${pairSymbol}`);
     }
     const lastPrice = ticker.last!;
-    if (pairSymbol.endsWith("/USD")) {
+    // Second condition is special case for binancecoinm
+    if (pairSymbol.endsWith("/USD") || pairSymbol.includes("/USD:")) {
```
Comment (rpiszczatowski) [inline packages/oracle-node/src/fetchers/ccxt/CcxtFetcher.ts:70]: Does it mean there was a bug around previously? I see that we already had tokens that matched the second condition...

File: packages/oracle-node/src/fetchers/evm-chain/uniswap-v3-on-chain/uniswap-v3-fetchers-config.json [hunk 2]
```diff
@@ -181,6 +181,54 @@
       "token1Decimals": 18,
       "fee": 3000,
       "pairedToken": "ETH"
+    },
```
Comment (rpiszczatowski) [inline packages/oracle-node/src/fetchers/evm-chain/uniswap-v3-on-chain/uniswap-v3-fetchers-config.json:184]: Maybe we should rename this file to uniswap-v3-ethereum-fetchers-config.json...

**Review by rpiszczatowski — DISMISSED** [review]

File: packages/oracle-node/src/fetchers/ccxt/CcxtFetcher.ts (hunk 1 above)
Comment (cehali) [reply packages/oracle-node/src/fetchers/ccxt/CcxtFetcher.ts:70]: Yes, `binancecoinm` was not working

**Review by rpiszczatowski — APPROVED** [review]

**Review by rpiszczatowski — APPROVED** [review]

**Review by rpiszczatowski — DISMISSED** [review]

**Review by rpiszczatowski — DISMISSED** [review]

File: packages/oracle-node/src/fetchers/ccxt/symbol-to-id/bitget.json [hunk 3]
```diff
@@ -5,5 +5,11 @@
...
   "AAVE": "AAVE/USDT",
   "STETH": "STETH/USDT",
-  "TRX": "TRX/USDT"
+  "TRX": "TRX/USDT",
+  "SHIB": "SHIB/USDT",
+  "PEPE": "PEPE/USDT",
+  "ARB": "ARB/USDT",
+  "OP": "OP/USDT",
+  "DYDX": "DYDX/USDT",
+  "AURA": "AURA1/USDT"
```
Comment (koonopek) [inline packages/oracle-node/src/fetchers/ccxt/symbol-to-id/bitget.json:14]: Typo?

File: packages/oracle-node/src/fetchers/evm-chain/optimism/velodrome/velodrome-fetchers-config.json [hunk 4]
```diff
@@ -51,11 +51,45 @@
   },
   "velodrome-on-chain-weth": {
     "ALETH": {
-      "poolAddress": "0x6fD5BEe1Ddb4dbBB0b7368B080Ab99b8BA765902",
+      "poolAddress": "0xa1055762336F92b4B8d2eDC032A0Ce45ead6280a",
```
Comment (koonopek) [inline packages/oracle-node/src/fetchers/evm-chain/optimism/velodrome/velodrome-fetchers-config.json:54]: Why do we change poolAddress?

**Review by koonopek — APPROVED** [review]

File: packages/oracle-node/src/fetchers/evm-chain/optimism/velodrome/velodrome-fetchers-config.json (hunk 4 above)
Comment (cehali) [reply packages/oracle-node/src/fetchers/evm-chain/optimism/velodrome/velodrome-fetchers-config.json:54]: They updated the pool from V1 to V2 and this is the new pool with more liquidity

File: packages/oracle-node/src/fetchers/ccxt/symbol-to-id/bitget.json (hunk 3 above)
Comment (cehali) [reply packages/oracle-node/src/fetchers/ccxt/symbol-to-id/bitget.json:14]: Nope: https://www.bitget.com/spot/AURA1USDT?type=spot

**Pull request merged**